packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.vue

```diff
@@ -76,3 +76,13 @@ watch(
     </template>
   </OnyxDataGridRenderer>
 </template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers.scss";
+
+@include layers.component {
+  .onyx-data-grid .onyx-table {
+    table-layout: fixed;
```

Comment:
Do we want to only change this for the data grid also make also the OnyxTable fixed by default?